src/Token/Token.js

```diff
@@ -23,12 +24,7 @@ Token.displayName = 'Token';
 
 Token.propTypes = {
     children: PropTypes.node,
-    className: PropTypes.string,
-    clickHandler: PropTypes.func
-};
-
-Token.propDescriptions = {
-    clickHandler: 'Callback function when user clicks on the component.'
```

Comment:
Needed to standardize on using `onClick` so it gets wired up properly if a `Token` is used as the `control` for a `Popover`.